share/picongpu/benchmarks/SPEC/include/picongpu/param/radiation.param

```diff
@@ -0,0 +1,207 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+#ifndef PARAM_RADFORMFACTOR
+#    define PARAM_RADFORMFACTOR radFormFactor_Gauss_spherical
+#endif
+
+#ifndef PARAM_RADWINDOWFUNCTION
+#    define PARAM_RADWINDOWFUNCTION radWindowFunctionTriplett
+#endif
+
+
+
+/*
+  radiation verbose level:
+  0=nothing, 1=physics, 2=simulation_state, 4=memory, 8=critical
+*/
+
+#define PIC_VERBOSE_RADIATION 3
+
+#include "picongpu/algorithms/Gamma.def"
+#include "picongpu/particles/manipulators/manipulators.def"
+#include "picongpu/plugins/radiation/VectorTypes.hpp"
+#include "picongpu/plugins/radiation/debug/PIConGPUVerboseLogRadiation.hpp"
+#include "picongpu/traits/attribute/GetMass.hpp"
+
+namespace picongpu
+{
+    namespace plugins
+    {
+        namespace radiation
+        {
+            namespace linear_frequencies
+            {
+                namespace SI
+                {
+                    constexpr float_64 omega_min = 0.0;
+                    constexpr float_64 omega_max = 1.06e16;
+                } // namespace SI
+
+                constexpr unsigned int N_omega = 1024; // number of frequencies
+            } // namespace linear_frequencies
+
+            namespace log_frequencies
+            {
+                namespace SI
+                {
+                    // plasma omega = sqrt( (electron density * (1.6e-19)^2) / (8.854e-12 * 9.11e-31) )
+                    //              = 1.78e14 1/s
+                    constexpr float_64 omega_pe = 1.78e14;
+                    constexpr float_64 omega_min = 0.1 * omega_pe;
+                    constexpr float_64 omega_max = 200 * omega_pe;
+                } // namespace SI
+
+                constexpr unsigned int N_omega = 256; // number of frequencies
+            } // namespace log_frequencies
+
+
+            namespace frequencies_from_list
+            {
+                /** path to text file with frequencies */
+                constexpr const char* listLocation = "/path/to/frequency.list";
+                constexpr unsigned int N_omega = 2048; // number of frequencies
+            } // namespace frequencies_from_list
+
+
+            namespace radiation_frequencies = log_frequencies;
+
+
+            namespace radiationNyquist
+            {
+                constexpr float_32 NyquistFactor = 0.5;
+            }
+
+            ///////////////////////////////////////////////////
+
+
+            // correct treatment of coherent and incoherent  radiation from macroparticles
+            /* Choose different form factors in order to consider different  particle shapes for radiation
+             *  - radFormFactor_CIC_3D ... CIC charge distribution
+             *  - radFormFactor_TSC_3D ... TSC charge distribution
+             *  - radFormFactor_PCS_3D ... PCS charge distribution
```

Comment:
note: the naming should be updated, I assume the radFormfactor should be named like our particle shapes.
This is nothing we should do in this PR because this affects many places in the code.